docker_images/fasttext/tests/test_api.py

```diff
@@ -11,6 +11,7 @@
 TESTABLE_MODELS: Dict[str, str] = {
     "text-classification": "osanseviero/fasttext_nearest",
     "feature-extraction": "osanseviero/fasttext_embedding",
+    "language-identification": "sheonhan/fasttext-language-identification",
```

Comment:
This is not needed. https://huggingface.co/sheonhan/fasttext-language-identification should also be tagged with `text-classification` as that's the primary task